Gems/Atom/RHI/Code/Platform/Linux/renderdoc_linux.cmake

```diff
@@ -6,24 +6,13 @@
 #
 #
 
-# Prevent bundling the renderdoc dll with a packaged title
-if(NOT LY_MONOLITHIC_GAME)
-    if(DEFINED ENV{"ATOM_RENDERDOC_PATH"})
-        set(RENDERDOC_PATH ENV{"ATOM_RENDERDOC_PATH"})
-    endif()
+set(PAL_TRAIT_BUILD_RENDERDOC_ENABLED FALSE)
+set(LY_RENDERDOC_PATH "/usr/local" CACHE PATH "Path to RenderDoc.")
```

Comment:
After this goes in, we may need to mention to the Atom folks a reminder that the ATOM_RENDERDOC_PATH environment variable isn't used any more (the code required this env variable previously)